foodsaving/users/api.py

```diff
@@ -42,6 +42,9 @@ def get_permissions(self):
             self.permission_classes = (AllowAny,)
         elif self.action in ('list', 'retrieve'):
             self.permission_classes = (IsAuthenticated,)
+        else:
+            # Issue 232 default permission classes should always be returned
+            self.permission_classes += self.__class__.permission_classes
```

Comment:
Verify_mail doesn't have get a user ID, therefore issameuser would throw an exception. I also understand better how permission classes are evaluated: from first to last, and only when necessary.
We have a misunderstanding about the default, it just means that every get_permissions() method should return self.permission_classes by default, to allow for the definition of other classes in the decorator. In this regard, the code above was just fine, but I didn't check the other APIViews yet. this is what the issue was about.
Maybe I should mark such codestyle-only issues, similar to as in the frontend repo.